scripts/elixir_github.sh

```diff
@@ -80,6 +80,12 @@ function upload_build {
   aws s3 cp *.zip "s3://s3.hex.pm/builds/elixir/${1}${2}.zip" --cache-control "public,max-age=3600" --metadata '{"surrogate-key":"builds","surrogate-control":"public,max-age=604800"}'
   fastly_purge $HEXPM builds
 
+  aws s3 cp s3://s3.hex.pm/builds/elixir/builds.txt builds.txt || true
+  touch builds.txt
+  echo -e "${1}${2} $(git rev-parse HEAD) $(date -u '+%Y-%m-%d %H:%M:%S')\n$(cat builds.txt)" > builds.txt
```

Comment:
I like that you added the date, but if we add it here we should it to the list of OTP builds also.

Reply:
I've added it there as well. Do you think just populating that over time is enough?